jujuclient/file.go

```diff
@@ -494,11 +495,42 @@ func (s *store) CurrentModel(controllerName string) (string, error) {
 			controllerName,
 		)
 	}
+
+	var controller bool
+	var modelNames []string
+	for ns := range controllerModels.Models {
```

Comment:
I assume ns is short for namespace here? `modelName` or smth would be better imo